pkg/costmodel/costmodel.go

```diff
@@ -1183,6 +1190,48 @@ func (cm *CostModel) GetNodeCost(cp costAnalyzerCloud.Provider) (map[string]*cos
 	return nodes, nil
 }
 
+// TODO: drop some logs
+func (cm *CostModel) GetLBCost(cp costAnalyzerCloud.Provider) (map[string]*costAnalyzerCloud.LoadBalancer, error) {
+	// for fetching prices from cloud provider
+	// cfg, err := cp.GetConfig()
+	// if err != nil {
+	// 	return nil, err
+	// }
+
+	servicesList := cm.Cache.GetAllServices()
+	loadBalancerMap := make(map[string]*costAnalyzerCloud.LoadBalancer)
+
+	// 1. need to check whether the service is a loadbalancer /
+	// 2. need to generate a unique key for this loadbalancer --> use name, which is unique across a namespace
+	// 3. need to check if key exists in servicesList and then populate loadBalancers with it
+
+	for _, service := range servicesList {
+		namespace := service.GetObjectMeta().GetNamespace()
+		name := service.GetObjectMeta().GetName()
+		// TODO: add clusterID to key? possible if called in ComputeCostData(), but what about metrics.go?
+		key := namespace + "," + name
+
+		// Does this identify ELB vs. ILB? Need to test the /api/allServices call with an ALB. Current work is on ELBs.
+		if service.Spec.Type == "LoadBalancer" {
+			// TODO: dynamically fetch based on cloud provider and region. Currently using hard-coded GCP us-central1 values.
+			loadBalancer := &costAnalyzerCloud.LoadBalancer{
```

Comment:
See above. I suggest you want a provider.LoadBalancer() that's a bit more generic than doing this here.